CMakeLists.txt

```diff
@@ -278,6 +278,8 @@ add_custom_target(
 set(VIAMCPPSDK_BOOST_VERSION_MINIMUM 1.71)
 set(VIAMCPPSDK_GRPC_VERSION_MINIMUM 1.30.2)
 set(VIAMCPPSDK_PROTOBUF_VERSION_MINIMUM 3.12.4)
+set(VIAMCPPSDK_XTENSOR_VERSION_MINIMUM 0.24.6)
+set(VIAMCPPSDK_XTENSOR_VERSION_MINIMUM 0.7.5)
```

Comment:
This should say XTL

Reply:
Also these versions need to be downgraded to match the current version available in sid.